BGL/package_info/BGL/dependencies

```diff
@@ -1,19 +1,11 @@
 Algebraic_foundations
-Arithmetic_kernel
 BGL
 Cartesian_kernel
 Circulator
-Distance_2
-Distance_3
-Filtered_kernel
 Hash_map
-Homogeneous_kernel
 Installation
-Intersections_2
-Intersections_3
 Interval_support
 Kernel_23
-Kernel_d
```

Comment:
If you run `make packages_dependencies` without openmesh found by CMake, then the dependencies for BGL are wrong.
The header `BGL/include/CGAL/boost/graph/properties_OpenMesh.h` includes `<CGAL/Exact_predicates_inexact_constructions_kernel.h>` and that triggers those dependencies to the CGAL core packages. Travis will probably complain.